### DISCUSSION TITLE
Incorrect academic status (Verified as teacher, stuck as a student)

### DISCUSSION THREAD

**Discussion by sondregronas** [community/community#41026] [GitHub Education] [answered]
Title.

When I'm on the page https://education.github.com/discount_requests/pack_application I get the following message on the Teacher application page: `You were last verified as a teacher on Oct 20, 2022. It is not necessary for you to re verify at this time.`. If I tick the box for student application the form does not say I'm verified (as a student). The form does not allow me to continue, as the buttons are simply greyed out.

However, I do _not_ have access to https://education.github.com/globalcampus/teacher as it always redirects me to the student page, and my  access is limited to the student resources. I've been told to look for a "Upgrade your academic organizations" section, on a dashboard I cannot get into.

I've sent a ticket regarding this, but was forwarded here. I am a educator and I wish to offer my students a better GitHub Classroom experience & explore what new possibilities that would ensue.

Thanks in advance.

**Comment by samus-aran** [comment, marked as answer]
Hi @sondregronas 👋 

Thanks for reaching out! We'll raise this with the team to assist with the incorrect status. We'll be in touch of any updates 👍

**Reply by samus-aran** [reply]
Just to confirm that we've been able to revoke the student status and you are able to re-submit as faculty (if you haven't done so already) 👍

**Reply by sondregronas** [reply]
Thank you! I've received your e-mails, and everything seems to be in working order now 😄

**Reply by zohim** [reply]
Same issue here, can you help?
Thanks

**Reply by zohim** [reply]
> Hi @sondregronas 👋
> 
> Thanks for reaching out! We'll raise this with the team to assist with the incorrect status. We'll be in touch of any updates 👍

Same issue here, can you help?
Thanks

**Reply by Shalinpatel1309** [reply]
same issue with mine

**Reply by shadow-jpg** [reply]
same with me, can you help please?

**Reply by meneerfaas** [reply]
I have got the same problem. I made a Github account last weekend and verified as a teacher, but I just received an e-mail welcoming me as a student to Global Campus. Quite strange.

**Reply by csta01** [reply]
Same issue

**Reply by Ozauri0** [reply]
Same issue, Has anyone been able to solve it?

**Reply by everynoanysome** [reply]
@samus-aran same issue here

**Reply by ghost** [reply]
Same issue

**Reply by s205225** [reply]
Same issue here
